StorySquad/Login/PinVC.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  PinVC.swift
+//  StorySquad
+//
+//  Created by Ian French on 6/7/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
+
+import UIKit
+
+class PinTextField: UITextField {
```

Comment:
It was a thought to create a class to reference later in the code instead of setting out each pin TF, but with 4 digits it was probably not necessary. I'll remove.